mkimage.sh

```diff
@@ -79,10 +79,13 @@ touch "$tarFile"
 	tar --numeric-owner -caf "$tarFile" -C "$rootfsDir" --transform='s,^./,,' .
 )
 
+cp -f 01_nodoc "$dir/01_nodoc"
+
 echo >&2 "+ cat > '$dir/Dockerfile'"
 cat > "$dir/Dockerfile" <<'EOF'
 FROM scratch
 ADD rootfs.tar.xz /
```

Comment:
`rootfs.tar.xz` already contains a lot of files in the exclude directories, can we filter them out too?

```
root@e259688ccb98:~# du -sh /usr/share/** | grep -E '(doc|man|groff|info|lintian|linda|locale)'
11M	/usr/share/doc
20K	/usr/share/doc-base
520K	/usr/share/info
144K	/usr/share/lintian
34M	/usr/share/locale
6.3M	/usr/share/man
3.3M	/usr/share/zoneinfo
```